servicebus/azure-spring-boot-starter-servicebus-jms/servicebus-jms-queue/pom.xml

```diff
@@ -5,31 +5,20 @@
     <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.5.4</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter-parent;external_dependency} -->
+    <groupId>com.azure.spring</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../../pom.xml</relativePath>
   </parent>
 
+  <artifactId>azure-spring-cloud-sample-servicebus-jms-queue</artifactId>
```

Comment:
Rename the sample artifact id with consistence with the new library artifact id .